content/v2.0/get-started.md

```diff
@@ -194,6 +195,100 @@ docker exec -it influxdb /bin/bash
 {{% /tab-content %}}
 <!--------------------------------- END Docker -------------------------------->
 
+<!-------------------------------- BEGIN kubernetes---------------------------->
+{{% tab-content %}}
+
+You can install InfluxDB in a local kubernetes environment with Minikube.
+
+1. Install minikube.
+
+    On macOS, run:
+
+    ```
+    brew install minikube
+    ```
+
+2. Start minikube:
+
+    ```
+    minikube start
+    ```
+
+3. Save the following YAML configuration file somewhere on your machine:
```

Comment:
...configuration file ~somewhere~ on your **local** machine.

Reply:
Also, maybe a quick description of what it includes: "This file will set up a namespace in your cluster named "monitoring", and create a Deployment for running and Service for connecting to InfluxDB.